File: mathx_robotics/tests/test_config.py
import os

def debug_visualize():
  return os.environ.get("DEBUG_VISUALIZE",None)=="1"

def debug_write():
  return os.environ.get("DEBUG_WRITE",None)=="1"

def debug_value(key,default_value):
  value=os.environ.get(key,str(default_value))
  return type(default_value)(value)

File: mathx_testing/src/mathx_testing/test_config.cc
#include <mathx_core/log.h>

#include <cstdlib>
#include <string>

namespace code {
namespace testing {

bool DebugVisualize() {
  char* val = std::getenv("DEBUG_VISUALIZE");
  // log("DEBUG_VISUALIZE=%s", val ? val : "(UNSET)");
  if (val && std::string(val) == "1") return true;
  return false;
}

bool DebugWrite() {
  char* val = std::getenv("DEBUG_WRITE");
  // log("DEBUG_WRITE=%s", val ? val : "(UNSET)");
  if (val && std::string(val) == "1") return true;
  return false;
}

}  // namespace testing
}  // namespace code
